SPECS/openvswitch/openvswitch.spec

```diff
@@ -79,216 +90,275 @@ Requires:       %{name} = %{version}-%{release}
 %description    doc
 It contains the documentation and manpages for openvswitch.
 
-%package -n	ovn-common
-Summary:        Common files for OVN
-Requires:       %{name} = %{version}-%{release}
-
-%description -n ovn-common
-It contains the common userspace components for OVN.
-
-%package -n	ovn-host
-Summary:        Host components of OVN
-Requires:       ovn-common = %{version}-%{release}
-
-%description -n ovn-host
-It contains the userspace components for OVN to be run on each hypervisor.
+%package        ipsec
+Summary:        Open vSwitch IPsec tunneling support
+License:        ASL 2.0
+Requires:       openvswitch 
+Requires:       python3-openvswitch 
+Requires:       strongswan
 
-%package -n	ovn-central
-Summary:        Central components of OVN
-Requires:       ovn-common = %{version}-%{release}
+%description    ipsec
+This package provides IPsec tunneling support for OVS tunnels.
 
-%description -n ovn-central
-It contains the user space components for OVN to be run on central host.
+%prep
+%autosetup -p1
 
-%package -n	ovn-controller-vtep
-Summary:        OVN VTEP controller binaries
-Requires:       ovn-common = %{version}-%{release}
+%build
+%configure \
+           --enable-ssl         \
+           --enable-shared      \
+		   --disable-static     \
+		   --with-dpdk=shared   \
+		   --enable-libcapng    \
+	       --with-pkidir=%{_sharedstatedir}/openvswitch/pki \
+           PYTHON3=%{__python3}
```

Comment:
Could you also run a linter against this spec to unify the usage of spaces vs tabs and for general clean-up?